public/index.html

```diff
@@ -29,19 +29,19 @@
     <div class="button-container">
       <button id="restart">Nouvelle partie</button>
       <div class="verify-container">
-        <button id="verify">Vérifier</button>
+        <button id="verify" type="submit">Vérifier</button>
```

Comment:
Actually this does not work because the button is not in a `<form> `

Reply:
removed